### PR TITLE
Adding announcement of Account Link extension

### PR DESCRIPTION
**Pull request by machuga**
Posted up for review / input
@diegopoza let me know what things I should move around or format differently. I'll make sure to run the linter at the end.

### CODE REVIEW

File: _posts/2017-09-15-easy-way-for-users-to-link-accounts.markdown [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+- 2017-08-11-how-identity-can-help-you-create-a-great-user-onboarding
+- 2017-07-21-the-role-of-identity-in-application-modernization
+---
+
+# An Easy Way for Users to Link Accounts
+
+A few months ago, Diego Poza wrote an article on [The 3 Best Ways To Create the Single Vision of A Customer](https://auth0.com/blog/the-three-best-ways-to-create-the-single-vision-of-a-customer/).
+In it he said the number one must-have is Centralized Identity Management. He 
+described that in Auth0, if a user signed in with their Facebook login, then 
+their Google account the next, they would have two accounts. Auth0 already
```
Comment (kmaida) [inline _posts/2017-09-15-easy-way-for-users-to-link-accounts.markdown]: `their Google account the next` -> "their Google account next" (remove the word `the`)

File: _posts/2017-09-15-easy-way-for-users-to-link-accounts.markdown [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+provides rules for allowing these accounts to be linked back together 
+automatically, but we're happy to announce a new option that gives the
+user a choice as to whether they'd like to link the accounts together.
```
Comment (kmaida) [inline _posts/2017-09-15-easy-way-for-users-to-link-accounts.markdown]: `accounts together` -> "accounts or not"

File: _posts/2017-09-15-easy-way-for-users-to-link-accounts.markdown [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+This happens as part of our Account Link extension. It's extremely simple to setup
```
Comment (kmaida) [inline _posts/2017-09-15-easy-way-for-users-to-link-accounts.markdown]: `setup` -> "set up" (verb context should be two words, nouns are one)

File: _posts/2017-09-15-easy-way-for-users-to-link-accounts.markdown [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+user a choice as to whether they'd like to link the accounts together.
+
+This happens as part of our Account Link extension. It's extremely simple to setup 
+and can save your users a lot of hassle.
+
+Let's walk through how this gets presented to your users to demonstrate how the
+functionality is helpful. Let's say one of your users has the email address
+`spencer@example.com`. They've used this email address to sign up for Facebook,
+Twitter, Google Apps, and your site. If your site offers all of these social
+sign in options, and database users, it can be understandably hard for a user
```
Comment (kmaida) [inline _posts/2017-09-15-easy-way-for-users-to-link-accounts.markdown]: `sign in options, and database` -> remove comma after "options"

File: _posts/2017-09-15-easy-way-for-users-to-link-accounts.markdown [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+to remember what account they used to sign up for your site. Like in our
+example above, we'll sign up that `spencer@example.com` signed up with their
```
Comment (kmaida) [inline _posts/2017-09-15-easy-way-for-users-to-link-accounts.markdown]: `we'll sign up that` -> "we'll assume that"

File: _posts/2017-09-15-easy-way-for-users-to-link-accounts.markdown [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+Facebook account originally, but now that they're back on the site they can't
+remember. They incorrectly guess they used Google to sign in.
+
+With the extension enabled, instead of being taken to your site with a new
+account, they are sent to a prompt from the extension, notifying them that they
+appear to already have an account registered with that email address on your
+site. The most prominent option is to continue linking the accounts. When
+**Continue** is clicked, the user will be redirected to login with the account
```
Comment (kmaida) [inline _posts/2017-09-15-easy-way-for-users-to-link-accounts.markdown]: `redirected to login with` -> "redirected to log in with" (verb form, two words)

File: _posts/2017-09-15-easy-way-for-users-to-link-accounts.markdown [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+that's already detected on your site. Assuming their authentication is
+successful, they'll be redirected to your site successfully as their original
+account, with the new account merged in as an identity of the original. This
+then, conveniently, lets them sign in with either option to the same account.
+
+![Image of extension page]()
+<!-- Image in docs going through code review -->
+
+But let's assume the opposite for a moment, that your user has intentionally
```
Comment (kmaida) [inline _posts/2017-09-15-easy-way-for-users-to-link-accounts.markdown]: `opposite for a moment,` -> change comma to a colon ":"

File: _posts/2017-09-15-easy-way-for-users-to-link-accounts.markdown [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+tried to create a second account with a different provider. Not a problem! They
+can simply click the small, diminished link saying, "I want to skip this and
+create a new account." option and they'll continue to your site logged in as a
```
Comment (kmaida) [inline _posts/2017-09-15-easy-way-for-users-to-link-accounts.markdown]: Remove the word `option`

File: _posts/2017-09-15-easy-way-for-users-to-link-accounts.markdown (hunk 5 above)
Comment (machuga) [reply _posts/2017-09-15-easy-way-for-users-to-link-accounts.markdown]: I am insanely impressed you were able to figure out what I meant there 😆

**Comment by machuga** [comment]
@kmaida Thanks for the thorough review! I've made the requested corrections.

**Comment by machuga** [comment]
@diegopoza the extension has been deployed to the gallery so this is safe to go out at your discretion.

**Comment by machuga** [comment]
Hold up for a bit - having build issues.

**Comment by machuga** [comment]
Ready to go!

**Pull request merged**